### ISSUE TITLE
Unable to install cryptonite-0.9 on windows ghc 7.10.2

### ISSUE THREAD

**Issue by katsuyas** [haskell-crypto/cryptonite#49]
$ cabal install cryptonite-0.8
will install it successfully, but

$ cabal install cryptonite-0.9
will generate an error. It seems that the error is reproducible regardless of using cygwin console or cmd.exe.
I have attached log file cryptonite-0.9.log
[cryptonite-0.9.txt](https://github.com/haskell-crypto/cryptonite/files/46273/cryptonite-0.9.txt)

Environment:
- Windows 7
- HaskellPlatform 7.10.2-a-i386
- ghc - 7.10.2

Can you please let me if I want to provide more info?

**Comment by centromere** [comment]
Hi @katsuyas. If #51 was applied, would this fix the problem?

**Comment by mpietrzak** [comment]
This is exactly the same error I got while trying to compile on arm and #51 fixed that for me.

**Comment by katsuyas** [comment]
@centromere
I will try it and let you know - I'm new to github, it should be simple to try in my local.

**Comment by katsuyas** [comment]
@centromere 
I've tested it locally. Looks working. Here is what I've done:

$ git clone https://github.com/haskell-crypto/cryptonite.git
$ cd cryptonite
$ git pull pull/51/head:master
$ cabal sandbox init
$ cabal update
$ cabal install -j --dependencies-only
$ cabal configure
$ cabal build

And it failed like before, but if I do 
$ cabal configure --flag='-support_blake2'
$ cabal build

then the build is successful.
So yes, I have confirmed the fix.
It might be part of your plan, but it would be nice that the disabling the feature is documented in the README.md file

Thanks,
